app/scripts/components/common/cookie-consent/index.tsx

```diff
@@ -42,67 +66,104 @@ export const CookieConsent = ({
     )}; path=/; expires=${closeConsent ? '0' : setCookieExpiration()}`;
   };
 
+  const setSessionData = () => {
+    if (typeof window !== 'undefined') {
+      const checkForSessionDate = window.sessionStorage.getItem(SESSION_KEY);
+      if (!checkForSessionDate) {
+        window.sessionStorage.setItem(SESSION_KEY, 'true');
+      }
+    }
+  };
+  useEffect(() => {
+    if (sessionStart !== 'true' && !cookieConsentResponded) {
+      setSessionData();
+      getCookie();
```

Comment:
nipick - the name of function can be clearer imo. getCookieis really not working as a getter, would there be any better name to reflect what it does?